### PR TITLE
Add perf limitation

### PR DESCRIPTION
**Pull request by 1e-to**
I made restrictions for all methods except string methods ljust and lower, @densmirn  where can I provide documentation for them?

### CODE REVIEW

**Comment by densmirn** [comment]
> I made restrictions for all methods except string methods ljust and lower, @densmirn where can I provide documentation for them?

https://github.com/IntelPython/sdc/pull/639
https://github.com/IntelPython/sdc/pull/700

File: sdc/datatypes/hpat_pandas_dataframe_rolling_functions.py [hunk 1]
```diff
@@ -575,6 +583,8 @@ def sdc_pandas_dataframe_rolling_var(self, ddof=1):
     """
     Limitations
     -----------
+    This function may reveal slower performance than Pandas* on user system. Users should exercise a tradeoff
```
Comment (PokhodenkoSA) [inline sdc/datatypes/hpat_pandas_dataframe_rolling_functions.py]: I think it is better to convert it to list.

File: sdc/datatypes/hpat_pandas_series_rolling_functions.py [hunk 2]
```diff
@@ -1154,6 +1154,8 @@ def hpat_pandas_series_rolling_var(self, ddof=1):
     """
     Limitations
     -----------
+    This function may reveal slower performance than Pandas* on user system. Users should exercise a tradeoff
```
Comment (PokhodenkoSA) [inline sdc/datatypes/hpat_pandas_series_rolling_functions.py]: I think it should be converted to list.

Comment (PokhodenkoSA) [reply sdc/datatypes/hpat_pandas_series_rolling_functions.py]: And in other places where Limitations consist of several statements they should be organized in list.

**Pull request merged**